### ISSUE TITLE
[Web] [Accessibility][11yMAS][Vision Impairment][MAS1.4.3] [Luminosity ratio is less than 4.5:1 for the “code. run. share” text .]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#706]
**User Experience:**
User with partial vision disability may not be able to clearly view the control or the text as the luminosity ratio is less than 4.5:1.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools Used:CCA
**Attachments:**
![mas1 4 3_ luminousity ratio](https://user-images.githubusercontent.com/33054846/32547781-a2412540-c4a9-11e7-904b-4b6e9ac977da.png)

**Repro Steps:**
1.	Open the URL: https://script-lab.azureedge.net/.
2.	Verify the luminosity ratio for the “code. run. share” text.

**Current Behavior:**
Luminosity ratio for the “code. run. share” text is 4.0:1 .

**Expected Behavior:**
Luminosity ratio for the “code. run. share” text should be equal or more than 4.5:1

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=1.4.3
